playhouse/tests/test_migrate.py

```diff
@@ -641,3 +641,6 @@ class MySQLMigrationTestCase(BaseMigrationTestCase, PeeweeTestCase):
     # MySQL does not raise an exception when adding a not null constraint
     # to a column that contains NULL values.
     _exception_add_not_null = False
+
+    def test_modify_not_null_foreign_key(self):
```

Comment:
Covers bug documented in mysql: https://bugs.mysql.com/bug.php?id=77591.

Travis CI currently only has 5.6.33 unless you do before_script hackery. Upstream chooses to simply omit this test, so I am doing the same.